client/blocks/term-form-dialog/index.jsx

```diff
@@ -52,14 +49,14 @@ class TermSelectorAddTerm extends Component {
 
 	static defaultProps = {
 		onSuccess: noop,
-		showDescriptionInput: false
+		showDescriptionInput: false,
+		showDialog: false
```

Comment:
Since `onClose` is not required, should we add a default `noop` prop for it here?